exonum-java-binding/time-service/README.md

```diff
@@ -0,0 +1,47 @@
+# Time oracle
+
+Java Binding Time Oracle provides proxy for [Exonum Time Service][exonum-time].
+This service allows to determine time, import it from the external world to the blockchain and keep
+its current value in the blockchain.
+
+## Usage
+
+Include `exonum-java-binding-time-service` (TODO: change the module name first) as a dependency in your `pom.xml`:
+
+``` xml
+    <dependency>
+      <groupId>com.exonum.binding</groupId>
+      <artifactId>exonum-java-binding-time-service</artifactId>
+      <version>0.4.0</version>
+    </dependency>
+```
+
+To use the time oracle, it needs to be enabled. To do that, include its name `time` in
+`ejb_app_services.toml` configuration file in the EJB App's directory with the following content:
+
+```toml
+services = ["time"]
+```
+
+See more information on built-in services [here][built-in-services].
+
+Below is an example of a service method that uses time oracle to return the consolidated time. Note that at the time when a new blockchain is launched, the consolidated time is unknown. In that case the result will not contain a value.
```

Comment:
I'd specify when it becomes known:

… is unkown. It is set when _y_, which normally happens within _x_  blocks.

y: the transactions from at least two thirds of validator nodes are processed?
x: several? a couple of? Is that correct there are no strict guarantees?

I’d look for exact 'x' and 'y' in the docs.